src/main/scala/chiseltest/backends/SimulatorInterface.scala

```diff
@@ -0,0 +1,30 @@
+package chiseltest.backends
+
+import firrtl.AnnotationSeq
+
+/** API to Simulator. */
+trait SimulatorInterface {
+
+  /** start time of this test. */
+  final val startTime: Long = System.nanoTime()
+
+  /** how many nano seconds has elapsed since this test start. */
+  def elapsedNanoSeconds(): Long = System.nanoTime() - startTime
```

Comment:
Should probably clarify this (by renaming) as walltime (to distinguish it from simulation time).
Also does it really need to be here? Unless it's meant to be a high accuracy API or something (and it doesn't seem like it), wouldn't it be cleaner to have whatever upper layer needs the performance data to handle the timing?

Reply:
maybe expose an API to return `wallTime` in second?

Reply:
So my argument is that this API doesn't belong here in the first place. Not that time isn't useful anywhere - but that timing code should go where it is used.

If you really feel that most SimulatorInterface use cases need timing data (which I don't think so), consider making a mixin trait that provides the timing data, while leaving SimulatorInterface as a bare essentials simulator interface API.